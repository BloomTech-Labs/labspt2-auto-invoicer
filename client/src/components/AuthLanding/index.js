import React from 'react';

const AuthLanding = props => {
  return (
    <ul className="auth-container">
      {/* <button className="authentication-btns" onClick={props.signUpModal}>
        Sign Up
      </button> */}
      <button className="authentication-btns" onClick={props.signInModal}>
        Sign In
      </button>
    </ul>
  );
};

export default AuthLanding;
