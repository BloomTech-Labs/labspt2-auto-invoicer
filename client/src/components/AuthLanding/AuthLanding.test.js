import React from "react";
import { render } from "react-testing-library";
import "jest-dom/extend-expect";
import { BrowserRouter as Router } from "react-router-dom";
import AuthLanding from "../AuthLanding";

describe("<AuthLanding />", () => {
  it("renders Auth Landing components", () => {
    const { asFragment } = render(
      <Router>
        <AuthLanding />
      </Router>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
