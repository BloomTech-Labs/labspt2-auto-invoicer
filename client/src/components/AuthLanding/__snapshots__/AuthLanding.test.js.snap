// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthLanding /> renders Auth Landing components 1`] = `
<DocumentFragment>
  <ul
    class="auth-container"
  >
    <button
      class="authentication-btns"
    >
      Sign In
    </button>
  </ul>
</DocumentFragment>
`;
